### DISCUSSION TITLE
What is the difference between linux-x64-static.tar and ubuntu-aarch64.tar?

### DISCUSSION THREAD

**Discussion by dmiepub** [PostgREST/postgrest#1913] [Q&A] [answered]
What is the difference between linux-x64-static.tar and ubuntu-aarch64.tar? Which is better for production usage on ununtu server?

postgrest-v8.0.0-linux-x64-static.tar.xz       3.09 MB
postgrest-v8.0.0-ubuntu-aarch64.tar.xz      13.4 MB

**Comment by steve-chavez** [comment, marked as answer]
Hey @dmiepub,

`linux-x64-static.tar` can be used on any linux distribution on an [x64 CPU](https://en.wikipedia.org/wiki/X86-64) (most common ones). You can check your CPU architecture with:

```bash
lscpu | grep Architecture
Architecture:        x86_64
```

If your Ubuntu is x86_64 then you should use this one.

`ubuntu-aarch64.tar` is for [ARM CPUs](https://en.wikipedia.org/wiki/AArch64), this binary only works for Ubuntu(maybe Debian/Raspbian as well).

```bash
lscpu | grep Architecture
Architecture:                    aarch64
```

**Reply by dmiepub** [reply]
Thank you @steve-chavez
